src/app/cli/src/coda.ml

```diff
@@ -502,18 +489,17 @@ let daemon logger =
          let get_port {Flag.Types.value; default; name} =
            or_from_config YJ.Util.to_int_option name ~default value
          in
-         let external_port = get_port external_port in
-         let client_port = get_port client_port in
+         let libp2p_port = get_port libp2p_port in
          let rest_server_port = get_port rest_server_port in
-         let discovery_port = get_port discovery_port in
+         let client_port = get_port client_port in
          let snark_work_fee_flag =
            let json_to_currency_fee_option json =
              YJ.Util.to_int_option json |> Option.map ~f:Currency.Fee.of_int
            in
            or_from_config json_to_currency_fee_option "snark-worker-fee"
              ~default:Cli_lib.Default.snark_worker_fee snark_work_fee
          in
-         let max_concurrent_connections =
+         let _max_concurrent_connections =
```

Comment:
Shouldn't this be removed if we aren't going to support it? Or, at the least, print a deprecation warning?